go.mod

```diff
@@ -4,7 +4,7 @@ go 1.17
 
 require (
 	github.com/aws/aws-sdk-go v1.42.0
-	github.com/aws/aws-sdk-go-v2 v1.11.2
+	github.com/aws/aws-sdk-go-v2 v1.16.2
```

Comment:
@MisterMX  did have an feeling what else changed for other resources ?

Reply:
No I don't. I just checked that I need at least `v1.20.0` of lambda to support `principalOrgId`.

But looking at the [Changelog](https://github.com/aws/aws-sdk-go-v2/blob/release-2022-04-19/CHANGELOG.md) there seem to be now breaking changes.